src/node_rc.cc

```diff
@@ -0,0 +1,198 @@
+#include "node_rc.h"
+#include "debug_utils-inl.h"
+#include "env-inl.h"
+#include "node_errors.h"
+#include "node_file.h"
+#include "node_internals.h"
+#include "simdjson.h"
+
+#include <functional>
+#include <map>
+#include <numeric>
+#include <string>
+
+namespace node {
+
+ConfigReader::ParseResult ConfigReader::ParseExperimentalObject(
+    simdjson::ondemand::object* experimental_object,
```

Comment:
@lemire ... I'm curious, does simdjson support jsonc? (JSON with comments). Would it be difficult to add support for jsonc if not? If we're going to use json as a configuration format, then supporting comments is ideal. (note that both deno and bun support comments in their respective formats)

Reply:
understood, ok thanks for the answer!